include/stdexec/execution.hpp

```diff
@@ -239,11 +243,9 @@ namespace stdexec {
           [[no_unique_address]] _BaseEnv __base_env_{};
 
           // Forward the receiver queries:
-          template <
-              __none_of<__tag_of<_WithIds>..., get_completion_signatures_t> _Tag,
-              same_as<__t> _Self,
-              class... _As>
-              requires __callable<_Tag, const __base_env_of<_Self>&, _As...>
+          template <__forwarding_query _Tag, same_as<__t> _Self, class... _As>
+              requires __none_of<_Tag, __tag_of<_WithIds>...> &&
+                __callable<_Tag, const __base_env_of<_Self>&, _As...>
```

Comment:
Nice catch, thanks.

Reply:
Thanks, but that was you in a prior commit 3weeks ago, I think.